### PR TITLE
add php8 support

### PR DESCRIPTION
**Pull request by mathieupetrini**
Add support for PHP 8.0

#422

### CODE REVIEW

**Comment by shmshd** [comment]
@stof, any update?

**Review by derrabus — APPROVED** [review]

**Comment by tarlepp** [comment]
Any update on this?

**Comment by VincentLanglet** [comment]
Friendly ping @stof 

Do you have some time to look at this PR ?
Are you looking for maintainer to help you ?

Take care

**Comment by shmshd** [comment]
@stof could you please accept this PR? This is the only library keeping most of us from upgrading our projects to PHP 8.0.

**Comment by deg-pl** [comment]
It's strange - there is ready PR, without any reaction.
I've just removed StofDoctrineExtensionsBundle from my projects and managed all functionality by myself. Now my projects are PHP8.0 compatibile... But it is not right way...

**Comment by stephanvierkant** [comment]
@derrabus Can you accept this PR or contact @stof?

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +17,7 @@ matrix:
     - php: 7.1
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.4
-      env: STABILITY=dev
+    - php: 8.0
```
Comment (Nyholm) [inline .travis.yml:20]: Could you please revert this change? 

It is not needed, but we should test dev-stability on PHP7.4

**Review by Nyholm — APPROVED** [review]
Im happy with this. Just a super small fix

File: .travis.yml (hunk 1 above)
Comment (mathieupetrini) [reply .travis.yml:21]: done

Comment (derrabus) [reply .travis.yml:21]: This line can be removed as well. You've added `8.0` in line 8 already, that should be enough.

**Comment by derrabus** [comment]
> @derrabus Can you accept this PR

I don't have permissions on this repo.

File: .travis.yml (hunk 1 above)
Comment (mathieupetrini) [reply .travis.yml]: done, thanks for review

**Review by derrabus — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Is it possible to tag a new release @stof ? :)

**Comment by stephanvierkant** [comment]
@VincentLanglet Stof hasn't replied for more than a month here. I've tried multiple ways to contact him. See #422 for more information. I even contacted someone from the Symfony Core Team (since symfony.com recommends this bundle), but they're not responsible for third-party bundles. That's why I propose to remove this bundle from the Symfony documentation: https://github.com/symfony/symfony-docs/pull/14909

**Comment by antishov** [comment]
@stephanvierkant hi, you're able to use my [fork](https://github.com/antishov/StofDoctrineExtensionsBundle)

**Comment by stephanvierkant** [comment]
Thanks! I see it's a popular fork, well done! I think it's in our shared interest to have a well-maintained and community-driven bundle, so feel free to create a PR in `symfony/symfony-docs` to replace Stof's bundle by your fork. And let's be clear about the maintainers, to prevent having the same issue (a bundle that depends on a single person) ever again.

**Comment by tarlepp** [comment]
I would recommend just to use that gedmo library directly - it's super easy to use with Symfony 4.x+ - https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/doc/symfony4.md
